ark/utils/deepcell_service_utils.py

```diff
@@ -0,0 +1,98 @@
+from ark.utils import io_utils
+from twisted.internet import reactor
+from kiosk_client import manager
+import os
+import glob
+from zipfile import ZipFile
+import warnings
+
+
+def create_deepcell_output(deepcell_input_dir, deepcell_output_dir, points=None,
+                           suffix='_feature_0', host='https://deepcell.org', job_type='multiplex'):
+    """ Handles all of the necessary data manipulation for running deepcell tasks.
+
+        Creates .zip files (to be used as input for DeepCell),
+        calls run_deepcell_task method,
+        and extracts zipped output files to the specified output location
+
+        Args:
+            points (list):
+                List of points in preprocessing pipeline. if None, all .tif files
+                in deepcell_input_dir will be considered as input points.
+            deepcell_input_dir (str):
+                Location of preprocessed files (assume deepcell_input_dir contains <point>.tif
+                for each point in points list)
+            suffix (str):
+                Suffix for DeepCell output filename. e.g. for pointX, DeepCell output
+                should be <pointX>+suffix.tif. Default: '_feature_0'
+            deepcell_output_dir (str):
+                Location to save DeepCell output (as .tif)
+            host (str):
+                Hostname and port for the kiosk-frontend API server
+                Default: 'https://deepcell.org'
+            job_type (str):
+                Name of job workflow (multiplex, segmentation, tracking)
+                Default: 'multiplex'
```

Comment:
You should put your args list in the order you specified your parameters in the function (so, `deepcell_input_dir`, `deepcell_output_dir`, `points`, etc.).

Reply:
Updated in the last commit